is/rules.go

```diff
@@ -6,122 +6,122 @@
 package is
 
 import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
 	"regexp"
 	"unicode"
 
-	"github.com/go-ozzo/ozzo-validation/v3"
 	"gopkg.in/asaskevich/govalidator.v9"
 )
 
 var (
 	// Email validates if a string is an email or not.
-	Email = validation.NewStringRule(govalidator.IsEmail, "must be a valid email address")
+	Email = validation.NewStringValidator(govalidator.IsEmail, "validation_email_invalid", "must be a valid email address")
```

Comment:
I think we do not need to `validation` prefix nor `invalid` postfix because the translator can append and prepend some strings. we just need to something like `invalid_email`

Reply:
The prefix is like a namespace to prevent potential conflict with other messages in an application.
I didn't say we need `invalid` suffix for string rules.

Reply:
You right, but I think namespacing can be handled by translators or any other function that want to use this error codes.

Reply:
How will a translator do namespacing?

Reply:
Translator gets the translation key(code prop in Error), so it can add every prefix that wants.
something like this:
```go
// user's translator pass error's code and params to translate function:
func translate(code string, params map[string]interface{})string {
    // User decide to has "v_" as vaildation error codes prefix.
    code:=fmt.Sprintf("v_%s",code)
   // Translate...
}